dev-vcs/fuel/fuel-1.0.1.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A Qt Fossil GUI"
+DESCRIPTION="Fuel is cross-platform GUI front-end for the excellent Fossil SCM \
+tool written in Qt. Fuel runs on Windows, OSX as well as various flavors of \
+Unix including Linux and FreeBSD."
+HOMEPAGE="https://fuel-scm.org/fossil/home/"
+COPYRIGHT="2018 Kostas Karanikolas"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://fuel-scm.org/files/releases/fuel-$portVersion.tar.gz"
+CHECKSUM_SHA256="4e463cb9f3cf9c31c7ae95ce76b558502c0b07e1dd4109da714997f0dc9a65db"
+PATCHES="fuel-$portVersion.patchset"
+ADDITIONAL_FILES="fuel.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	fuel$secondaryArchSuffix = $portVersion
+	app:fuel
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:fossil
+	lib:libcrypto$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	qmake fuel.pro
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p "$appsDir"
```

Comment:
You don't need quotes here and below.

Reply:
We **_currently_** don't need them. But if someone chooses to use a `TREE_PATH` with a space character, and haikuporter's `--nochroot` switch is implemented in a way that allows to build recipes, then the quotes would be necessary. (Yeah, I know, this will probably never happen because a runtimes compiled outside of the chroot would need to be changed to make their paths correct, but this is IMHO not a good reason to not use the quotes here.)
To summarise:
1) We have lots of quotes in places where they will never be required.
2) In places like here, were it would be nice to start using them, I don't see why we would drop them.

I've given up suggesting this to others. I use the quotes in the recipes I edit. If this can serve as a model to other contributors, then I'm happy. We don't need to discourage the use of the quotes imo.